src/docs/product/alerts/create-alerts/issue-alert-config.mdx

```diff
@@ -92,9 +92,14 @@ Learn more about [routing alerts with integrations](/product/alerts/create-alert
 
 [Issue owners](/product/issues/issue-owners/) can receive notifications by email or Slack depending on the [notification settings](/product/alerts/notifications/notification-settings/) of the issue owner when an alert is triggered.
 
-If an issue owner is not configured or not found, either the notification won’t be sent or it’ll be sent to all project members, depending on the following setting in **[Project] > Settings > Issue Owners**.
+If an issue owner is not configured or not found, the notification will be sent to the teams or members specified in the fallback setting:
```

Comment:
So I've just realized that while we're calling this the "fallback" rule internally, there's nothing that says this in the UI, so I think we want to make sure we call this the "fallback notification setting" going forward so it's super clear for users. I also think we should put the new image in the old image location so it's clearer that this is the setting we're talking about before introducing the options in it.
```suggestion
If an issue owner is not configured or not found, the notification will be sent to the teams or members specified in the fallback notification setting:

![Issue owners with notify all project members selected.](issue_owners_all_members.png)

```

Reply:
@scttcper - are you good with this change?